addons/transparent-orphans/addon.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Transparent orphan blocks",
+  "description": "Makes any blocks that are orphaned (unused with no parent block) transparent.",
+  "tags": ["editor"],
+  "settings": [
+    {
+      "name": "Transparency",
+      "id": "transparency",
+      "type": "integer",
+      "min": 0,
+      "max": 100,
+      "default": 50
+    }
+  ],
```

If you want, you can add credits like this:
```suggestion
  ],
  "credits": {
    "name": "ErrorGamer200"
    "link": "https://github.com/ErrorGamer2000"
  },
```
(you may also want to credit Colaber if you used some of his code)

---

(Credits is an array of objects)

---

I didn't and I know how to add credits, just forgot...